clang/test/Analysis/int128-nocrash.c

```diff
@@ -0,0 +1,10 @@
+// RUN: %clang_analyze_cc1 -analyzer-checker=optin.portability.UnixAPI \
+// RUN:    -triple x86_64-pc-linux-gnu -x c %s
+
+// Don't crash!
+// expected-no-diagnostics
+const __int128_t a = ( ((__int128_t)1) << 64 | 1);
```

Comment:
Why did you add the "`| 1`"? Does the test work without it?

Reply:
Hmm, I added this for no particular reason at the time. I'll try without it. I could not find a numeric literal suffix for a 128 bit number, this case was originally found by multiplying 2 64-bit numbers into a 128-bit number, and I simplified that.